### ISSUE TITLE
cmake-js No such file or directory

### ISSUE THREAD

**Issue by alecive** [robotology/yarp.js#11]
I apparently installed `cmake-js`:
```
[alecive@malakim]$ sudo npm install -g cmake-js
npm WARN deprecated npmconf@2.1.2: this package has been reintegrated into npm and is now out of date with respect to npm
/usr/bin/cmake-js -> /usr/lib/node_modules/cmake-js/bin/cmake-js
cmake-js@3.2.2 /usr/lib/node_modules/cmake-js
├── is-iojs@1.1.0
├── splitargs@0.0.7
├── url-join@0.0.1
├── semver@5.3.0
├── debug@2.2.0 (ms@0.7.1)
├── which@1.2.10 (isexe@1.1.2)
├── bluebird@2.10.2
├── memory-stream@0.0.3 (readable-stream@1.0.34)
├── yargs@3.32.0 (decamelize@1.2.0, y18n@3.2.1, camelcase@2.1.1, window-size@0.1.4, cliui@3.2.0, os-locale@1.4.0, string-width@1.0.2)
├── npmconf@2.1.2 (ini@1.3.4, uid-number@0.0.5, inherits@2.0.1, once@1.3.3, semver@4.3.6, osenv@0.1.3, nopt@3.0.6, config-chain@1.1.10, mkdirp@0.5.1)
├── npmlog@1.2.1 (ansi@0.3.1, gauge@1.2.7, are-we-there-yet@1.0.6)
├── fs-extra@0.16.5 (jsonfile@2.3.1, graceful-fs@3.0.9, rimraf@2.5.4)
├── tar@1.0.3 (block-stream@0.0.9, inherits@2.0.1, fstream@1.0.10)
├── unzip@0.1.11 (setimmediate@1.0.4, readable-stream@1.0.34, pullstream@0.4.1, match-stream@0.0.2, binary@0.3.0, fstream@0.1.31)
├── request@2.74.0 (aws-sign2@0.6.0, tunnel-agent@0.4.3, forever-agent@0.6.1, oauth-sign@0.8.2, caseless@0.11.0, is-typedarray@1.0.0, stringstream@0.0.5, aws4@1.4.1, isstream@0.1.2, json-stringify-safe@5.0.1, extend@3.0.0, tough-cookie@2.3.1, qs@6.2.1, node-uuid@1.4.7, combined-stream@1.0.5, mime-types@2.1.11, form-data@1.0.0-rc4, bl@1.1.2, hawk@3.1.3, http-signature@1.1.1, har-validator@2.0.6)
├── lodash@3.10.1
└── traceur@0.0.111 (commander@2.9.0, semver@4.3.6, rsvp@3.2.1, glob@5.0.15, source-map-support@0.2.10)
```
Successfully configured npm:
```
(~/code/yarp.js) (master) 
[alecive@malakim]$ sudo npm install
npm WARN cannot run in wd YarpJS@1.0.0 cmake-js (wd=/home/alecive/code/yarp.js)
```
But `cmake-js` is not found:
```
(~/code/yarp.js) (master) 
[alecive@malakim]$ cmake-js
: No such file or directory
```

**Comment by cciliber** [comment]
Thanks! The problem is due to an issue with cmake-js line endings in the last version of cmake-js. Thanks to you I opened an issue on cmake-js (see cmake-js/cmake-js#51). 

However I have improved the installation process for yarp.js. You don't need to do a global install of cmake-js anymore (see the README). Just install node and yarp and then run
```
$> sudo npm install
$> ./node_modules/cmake-js/bin/cmake-js
```
In this way you are calling your local copy of cmake-js. The local version of cmake-js is specified in the yarp.js `package.json` file. Until cmake-js is not fixed, I downgraded to the previous cmake-js version (cmake-js@3.2.1).

I tested it on both Linux (Ubuntu 14.04) and OSX (El Capitan).

Again, thanks for raising the issue!
